services/base-service/models/grant_role/model.js

```diff
@@ -97,9 +97,14 @@ class GrantRoleModel extends BaseModel {
         }
       `
     };
-    resp = await finApi.patch(options);
+    const api_resp = await finApi.patch(options);
 
-//    console.log({ options, id, patch, expertId, resp });
+    if (api_resp.last.statusCode != 204) {
```

Comment:
# Example 2 #
This example examines the result of a `fetch` between internal components. Since `fetch`  doesn't throw errors on 50X responses, then we need  to add logic, our application however *does* throw and error.  In this case, though we know that the actual fetch error was, since this is all internal, this is just a `500` to the user as well.